File: Java/Modul2_Percobaan4_Kel46.java
import java.util.Scanner;

public class Modul2_Percobaan4_Kel46 {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Masukan Angka 1-2 yang di inginkan = ");
        int i = sc.nextInt();
        switch(i){
        case(1):System.out.println("Angka yang anda masukan bernilai A");break;
        case(2):System.out.println("Angka yang anda masukan bernilai B");break;
        default:System.out.println("Angka yang anda masukan tak bernilai A ataupun B");break;
        }
        System.out.println("Program Selesai");
    }
}

File: CSharp/Modul2_Percobaan4_Kel46.cs
using System; 
  
namespace Modul2 { 
      
    class Hello { 
          
        static void Main(string[] args) { 
              
            string nilai;
            Console.Write("Masukan Nilai (A-E) :");
            nilai = Console.ReadLine();

            switch (nilai)
            {
                case "A":
                    Console.WriteLine("Nilai Anda\t: {0} (80 - 100)", nilai);
                    Console.ReadLine();break;
                case "B":
                    Console.WriteLine("Nilai Anda\t: {0} (80 - 100)", nilai);
                    Console.ReadLine(); break;
                case "C":
                    Console.WriteLine("Nilai Anda\t: {0} (80 - 100)", nilai);
                    Console.ReadLine(); break;
                case "D":
                    Console.WriteLine("Nilai Anda\t: {0} (80 - 100)", nilai);
                    Console.ReadLine(); break;
                case "E":
                    Console.WriteLine("Nilai Anda\t: {0} (80 - 100)", nilai);
                    Console.ReadLine(); break;
                default:
                    Console.WriteLine("Nilai yang Anda Masukan SALAH");
                    Console.ReadLine();break;
            }

        } 
    } 
} 